composer.json

```diff
@@ -54,7 +54,8 @@
     "pear/Auth_SASL": "1.1.0",
     "pear/Net_SMTP": "1.6.*",
     "pear/Net_socket": "1.0.*",
-    "civicrm/civicrm-setup": "~0.2.0"
+    "civicrm/civicrm-setup": "~0.2.0",
+    "guzzlehttp/guzzle": "^6.3"
```

Comment:
I wanted to do a quick sanity check to ensure that we wouldn't be throwing kink in D8 support. This looks OK -- one should be able to satisfy this constraint as well as all these variants from D8:

* Drupal 8.0.x: `"guzzlehttp/guzzle": "~6.1",`
* Drupal 8.1.x: `"guzzlehttp/guzzle": "^6.2.1",`
* ...
* Drupal 8.6.x: `"guzzlehttp/guzzle": "^6.2.1",`